eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -160,10 +159,14 @@ object Commitments {
 
     // a node cannot spend pending incoming htlcs, and need to keep funds above the reserve required by the counterparty, after paying the fee
     // we look from remote's point of view, so if local is funder remote doesn't pay the fees
-    val fees = if (commitments1.localParams.isFunder) commitTxFee(commitments1.remoteParams.dustLimit, reduced) else Satoshi(0)
-    val missing = reduced.toRemote.truncateToSatoshi - commitments1.remoteParams.channelReserve - fees
-    if (missing < Satoshi(0)) {
-      return Left(InsufficientFunds(commitments.channelId, amount = cmd.amount, missing = -missing, reserve = commitments1.remoteParams.channelReserve, fees = fees))
+    val fees = Transactions.commitTxFee(commitments1.remoteParams.dustLimit, reduced)
+    val missingLocal = reduced.toRemote.truncateToSatoshi - commitments1.remoteParams.channelReserve - fees
```

Comment:
I don't think that's correct, you make it seem like both sides are paying the commitment fee.